Generated code needs to use full qualification.

### Description

Please see https://github.com/apple/swift/issues/68105

### Expected behavior

_No response_

### Actual behavior

_No response_

### Steps to reproduce

_No response_

### Swift Package Manager version/commit hash

_No response_

### Swift & OS version (output of `swift --version ; uname -a`)

_No response_

I see this after resolving btw (when using 5.9):

```
Building for debugging...
/Users/neonacho/Downloads/y/Sources/y/y.swift:13:40: error: ambiguous use of 'fatalError'
  if CommandLine.arguments.count < 0 { fatalError("BOOM") }
                                       ^
/Users/neonacho/Downloads/y/Sources/y/y.swift:3:6: note: found this candidate
func fatalError(
     ^
Swift.fatalError:1:13: note: found this candidate
public func fatalError(_ message: @autoclosure () -> String = String(), file: StaticString = #file, line: UInt = #line) -> Never
            ^
error: fatalError
```

But changing it to `y.fatalError` makes it work.

--- a/Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift
+++ b/Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift
@@ -353,7 +353,8 @@ public final class SwiftTargetBuildDescription {
                     let preferredBundle = Bundle(path: mainPath)
 
                     guard let bundle = preferredBundle ?? Bundle(path: buildPath) else {
-                        fatalError("could not load resource bundle: from \\(mainPath) or \\(buildPath)")
+                        // Users can write a function called fatalError themselves, we should be resilient against that.
+                        Swift.fatalError("could not load resource bundle: from \\(mainPath) or \\(buildPath)")
                     }
 
                     return bundle

